php-practice_1.php

```diff
@@ -1,41 +1,161 @@
 <?php
 // Q1 変数と文字列
+$name = '安田';
 
+$newname ='私の名前は「' . $name . '」です。';
+echo  $newname;
 
 // Q2 四則演算
+$num  =  5* 4;
+echo  $num;
 
+echo  "\n";
+
+$num  /=  2;
+echo  $num;
 
 // Q3 日付操作
+date_default_timezone_set('Asia/Tokyo');
 
+echo '現在時刻は、' . date("Y年m月d日 H時i分s秒") . 'です。';
 
 // Q4 条件分岐-1 if文
-
+$device = '';//windows、mac、ipadなど入れてみて下さい！
+if ($device === 'windows') {
+    echo '使用のOSは、windowsです。';
+} else {
+  if ($device === 'mac') {
+    echo '使用のOSは、macです。';
+} else {
+    echo 'どちらでもありません。';
+       }
+}
 
 // Q5 条件分岐-2 三項演算子
+$age = '';//年齢を入れる。
+$message = ($age < 18) ? '未成年です。' : '成人です。';
 
+echo $message;
 
 // Q6 配列
-
+$prefecture = ['埼玉県' , '茨城県' , '栃木県' , '千葉県' , '長野県' , '神奈川県' , '東京都'];
+echo ''. $prefecture[2] . 'と' . $prefecture[3] . 'は関東地方の都道府県です。'
 
 // Q7 連想配列-1
-
+$prefectures = [
+  '東京都' => '新宿区',
+  '神奈川県' => '横浜市',
+  '千葉県' => '千葉県',
+  '埼玉県' => 'さいたま市',
+  '栃木県' => '宇都宮市',
+  '群馬県' => '前橋市',
+  '茨城県' => '水戸市'
+];
+
+echo $prefectures['東京都'],"\n";
+echo $prefectures['神奈川県'],"\n";
+echo $prefectures['千葉県'],"\n";
+echo $prefectures['埼玉県'],"\n";
+echo $prefectures['栃木県'],"\n";
+echo $prefectures['群馬県'],"\n";
+echo $prefectures['茨城県']
 
 // Q8 連想配列-2
-
+$prefectures = [
+  '東京都' => '新宿区',
+  '神奈川県' => '横浜市',
+  '千葉県' => '千葉県',
+  '埼玉県' => 'さいたま市',
+  '栃木県' => '宇都宮市',
+  '群馬県' => '前橋市',
+  '茨城県' => '水戸市'
+];
+
+foreach ($prefectures as $key => $value) {
+    if ($key === '埼玉県') {
+       echo $key . 'の県庁所在地は' . $value . 'です。';
+    }
+}
 
 // Q9 連想配列-3
-
+$prefectures = [
+  '東京都' => '新宿区',
+  '神奈川県' => '横浜市',
+  '千葉県' => '千葉県',
+  '埼玉県' => 'さいたま市',
+  '栃木県' => '宇都宮市',
+  '群馬県' => '前橋市',
+  '茨城県' => '水戸市',
+];
+
+$prefectures['愛知県']='名古屋';
+$prefectures['大阪府']='大阪市';
```

Comment:
@yuma-yasuda0216 
可読性（読みやすさ）をあげるために半角スペース開けましょう！
他にも同じような箇所あれば修正お願いします！

```suggestion
$prefectures['愛知県'] = '名古屋';
$prefectures['大阪府'] = '大阪市';
```